service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/bucket/StoragetransferConfiguration.java

```diff
@@ -0,0 +1,54 @@
+package bio.terra.workspace.service.resource.controlled.flight.clone.bucket;
+
+import com.google.api.client.googleapis.util.Utils;
+import com.google.api.services.storagetransfer.v1.Storagetransfer;
+import com.google.api.services.storagetransfer.v1.StoragetransferScopes;
+import com.google.auth.http.HttpCredentialsAdapter;
+import com.google.auth.oauth2.GoogleCredentials;
+import java.io.IOException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.BeanCreationException;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+@Configuration
+public class StoragetransferConfiguration {
```

Comment:
Given that this is making globally available Beans, I thnk this file should be next to other `@Configuration`s in `bio/terra/workspace/app/configuration` instead of deeply nested inside  `workspace/service`